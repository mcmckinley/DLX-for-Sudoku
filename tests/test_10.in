1  0  0  0   0  0  12 11  4  0  7  9   0  6  0  0  
0  0  12 0   10 0  14 0   16 0  1  0   5  0  0  0  
0  0  0  0   0  0  16 0   0  0  0  11  14 0  2  4  
0  0  4  0   8  0  0  0   0  0  13 0   0  0  15 0  

14 0  0  4   0  16 9  0   0  13 0  3   0  15 0  0  
13 3  2  0   0  0  0  12  9  0  14 6   0  0  0  0  
10 0  0  9   7  0  0  13  0  0  11 16  0  0  8  5  
0  1  0  0   2  0  11 0   0  5  0  0   0  0  9  14  

0  0  0  7   0  0  0  0   0  15 0  0   0  4  10 8  
16 15 0  0   0  0  10 8   0  0  0  13  6  0  7  11  
0  0  10 11  0  0  0  0   12 0  0  0   9  0  14 13  
9  0  0  0   14 0  13 0   0  11 0  0   0  0  16 12  

0  0  0  0   0  0  6  0   0  0  0  0   0  0  13 0  
0  0  0  8   13 11 0  14  0  7  15 0   0  0  4  0  
2  13 0  1   15 0  0  0   14 8  0  0   11 10 0  16  
15 0  5  0   0  8  0  16  0  0  6  0   0  1  0  0  