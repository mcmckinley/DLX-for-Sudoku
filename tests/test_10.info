Hard 16x16 B