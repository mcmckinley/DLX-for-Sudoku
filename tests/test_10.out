1  16 8  15  5  2  12 11  4  14 7  9   13 6  3  10 
6  7  12 2   10 13 14 4   16 3  1  15  5  8  11 9  
5  9  3  13  6  1  16 15  8  12 10 11  14 7  2  4  
11 14 4  10  8  9  7  3   5  6  13 2   12 16 15 1  

14 12 11 4   1  16 9  5   7  13 8  3   10 15 6  2  
13 3  2  5   4  15 8  12  9  10 14 6   16 11 1  7  
10 6  15 9   7  14 3  13  2  1  11 16  4  12 8  5  
8  1  7  16  2  10 11 6   15 5  12 4   3  13 9  14 

3  2  13 7   11 12 5  9   6  15 16 14  1  4  10 8  
16 15 14 12  3  4  10 8   1  9  5  13  6  2  7  11 
4  8  10 11  16 6  15 1   12 2  3  7   9  5  14 13 
9  5  1  6   14 7  13 2   10 11 4  8   15 3  16 12 

7  4  9  3   12 5  6  10  11 16 2  1   8  14 13 15 
12 10 16 8   13 11 1  14  3  7  15 5   2  9  4  6  
2  13 6  1   15 3  4  7   14 8  9  12  11 10 5  16 
15 11 5  14  9  8  2  16  13 4  6  10  7  1  12 3  
